File: src/com/pokosho/dao/Reply.java
package com.pokosho.dao;

import net.java.ao.Entity;
import net.java.ao.schema.NotNull;
import net.java.ao.schema.PrimaryKey;

public interface Reply extends Entity {
	@PrimaryKey
	@NotNull
	public Long getUser_ID();
	public void setUser_ID(Long userID);

	public Long getTweet_ID();
	public void setTweet_ID(Long tweetID);

	public void setTime(Integer t);
	public Integer getTime();
}

File: src/main/kotlin/com/pokosho/dao/Reply.kt
package com.pokosho.dao

import net.java.ao.RawEntity
import net.java.ao.schema.NotNull
import net.java.ao.schema.PrimaryKey

interface Reply : RawEntity<Int> {
    @get:PrimaryKey
    @get:NotNull
    var user_ID: Long?

    var tweet_ID: Long?
    var time: Int?
}
